NuKeeper.Abstract/Configuration/AuthSettings.cs

```diff
@@ -1,10 +1,10 @@
 using System;
 
-namespace NuKeeper.Configuration
+namespace NuKeeper.Abstract.Configuration
```

Comment:
In your comment you referred to this as `Abstractions`, I think `NuKeeper.Abstractions` a more suitable name than `NuKeeper.Abstract` and I've seen it in other libraries, not sure what everyone else feels about this.

Reply:
I'm not tied to either... If you would like me to change it i can